sandbox/gas/tube/go

```diff
@@ -134,9 +134,9 @@ class SodtubeMesher(object):
 def generate_bcmap():
     # BC map.
     bcmap = {
-        'wall': (sc.bctregy.GasWall, {},),
-        'left': (sc.bctregy.GasNonrefl, {},),
-        'right': (sc.bctregy.GasNonrefl, {},),
+        b'wall': (sc.bctregy.GasWall, {},),
+        b'left': (sc.bctregy.GasNonrefl, {},),
+        b'right': (sc.bctregy.GasNonrefl, {},),
```

Comment:
python2 to 3 compatibility issue. Make sure bytes string be used to match the desired elements of boundary condition mapper.